jina/orchestrate/flow/base.py

```diff
@@ -861,6 +887,7 @@ def add(
               - To access specified gpus based on device id, use `--gpus device=[YOUR-GPU-DEVICE-ID]`
               - To access specified gpus based on multiple device id, use `--gpus device=[YOUR-GPU-DEVICE-ID1],device=[YOUR-GPU-DEVICE-ID2]`
               - To specify more parameters, use `--gpus device=[YOUR-GPU-DEVICE-ID],runtime=nvidia,capabilities=display
+        :param grpc_metadata: The metadata to be passed to the gRPC request. This argument only applies for external Deployments.
```

Comment:
should this be a `client.post` argument instead of a Client instance argument?

Reply:
This PR is not related to `client.post`. But, in the current `client.post`, `metadata` is used as its argument, rather than `Client` instantce argument.